src/SFA.DAS.LevyTransferMatching.Web/StartupExtensions/AddEmployerAccountsApiExtensions.cs

```diff
@@ -1,14 +1,28 @@
-﻿using Microsoft.Extensions.DependencyInjection;
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
 using SFA.DAS.EmployerAccounts.Api.Client;
 using SFA.DAS.LevyTransferMatching.Infrastructure.Stubs;
+using EmployerAccountsApiClient = SFA.DAS.LevyTransferMatching.Infrastructure.Api.EmployerAccountsApiClient;
 
 namespace SFA.DAS.LevyTransferMatching.Web.StartupExtensions
 {
     public static class AddEmployerAccountsApiExtensions
     {
-        public static IServiceCollection AddEmployerAccountsApi(this IServiceCollection services)
+        public static IServiceCollection AddEmployerAccountsApi(this IServiceCollection services, IConfiguration config, IWebHostEnvironment environment)
         {
-            services.AddSingleton<IEmployerAccountsApiClient, StubEmployerAccountsApiClient>();
+            var useStub = config.GetValue<bool>("UseEmployerAccountApiStub");
```

Comment:
Do we need the employer account stub now we can use a local CosmosDb. Can the stub code be removed?